application/common/models/User.php

```diff
@@ -343,6 +343,55 @@ private function savePassword()
         return true;
     }
 
+    public static function getExpiringUsers($criteria): array
```

Comment:
Is the goal of `getExpiringUsers(...)` to get users that will expire within a day? The desired timeframe seemed ambiguous to me, so I wasn't sure.